### PR TITLE
[EventHubs] Preview 5

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

File: sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md [hunk 1]
```diff
@@ -110,20 +110,19 @@ Refer to [Logging](#logging) to enable loggers for related libraries.
 ## Next steps
 
 ### Examples
-- [./examples/eventprocessor/event_processor_blob_storage_example.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/examples/event_processor_blob_storage_example.py) - event processor with blob partition manager example
+- [./samples/event_processor_blob_storage_example.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/samples/event_processor_blob_storage_example.py) - EventHubConsumerClient with blob partition manager example
 
 ### Documentation
 
-Reference documentation is available at https://azure.github.io/azure-sdk-for-python/ref/azure.eventhub.extensions.html.
+Reference documentation is available at https://azuresdkdocs.blob.core.windows.net/$web/python/azure-eventhub/5.0.0b5/azure.eventhub.extensions.html
```
Comment (lmazuel) [inline sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md]: Extensions being empty, this link will not exits. What do you wanted to show here?

File: sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/setup.py [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
 exclude_packages = [
         'tests',
-        'examples',
+        'samples',
```
Comment (lmazuel) [inline sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/setup.py]: Folder is still called examples on the branch.
Also, your MANIFEST.in should include samples + tests (look for exampke in azure-core)

File: sdk/eventhub/azure-eventhubs-checkpointstoreblob/MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,4 @@
+include *.md
+include azure/__init__.py
+include azure/eventhub/__init__.py
+include azure/eventhub/extensions/__init__.py
```
Comment (lmazuel) [inline sdk/eventhub/azure-eventhubs-checkpointstoreblob/MANIFEST.in]: must include samples and tests (look for example in azure-core file)

File: sdk/eventhub/azure-eventhubs/HISTORY.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
 # Release History
 
+## 2019-11-04 5.0.0b5
+
+**Breaking changes**
+
+- `EventHubClient` has been split into two separate clients: `EventHubProducerClient` and `EventHubConsumerClient`.
```
Comment (lmazuel) [inline sdk/eventhub/azure-eventhubs/HISTORY.md]: Please mention all the module which are not plublic anymore (see release note of azure-core 1.0.0b4 for example)

File: sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md (hunk 1 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md]: I have updated the link , pointing to the `PartitionManager` in eventhub docs

File: sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/setup.py (hunk 2 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/setup.py]: sample directory name has been changed, also updated MANIFEST.in as required

File: sdk/eventhub/azure-eventhubs-checkpointstoreblob/MANIFEST.in (hunk 3 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs-checkpointstoreblob/MANIFEST.in]: updated

File: sdk/eventhub/azure-eventhubs/HISTORY.md (hunk 4 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/HISTORY.md]: We have only removed `EventHubClient`, `EventHubProducerClient`, `EventHubConsumerClient`.

File: sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md (hunk 1 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md]: I think it's due to the github that can't show the changes.

I checked files in the branch and see the links are up to date.

[README of azure-eventhubs-checkpointstoreblob](
https://github.com/Azure/azure-sdk-for-python/blob/feature/eventhubs_preview5/sdk/eventhub/azure-eventhubs-checkpointstoreblob/README.md)

[README of azure-eventhubs-checkpointstoreblob-aio](
https://github.com/Azure/azure-sdk-for-python/blob/feature/eventhubs_preview5/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/README.md)

File: sdk/eventhub/azure-eventhubs/HISTORY.md (hunk 4 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/HISTORY.md]: right, I've updated the changelog.

**Pull request merged**